frontend/src/features/properties/hooks/useParcelLayerData.tsx

```diff
@@ -125,13 +122,9 @@ const useParcelLayerData = ({ formikRef, parcelId, nameSpace }: IUseParcelLayerD
   );
   const { getByType } = useCodeLookups();
   const [showOverwriteDialog, setShowOverwriteDialog] = useState(false);
+
   useDeepCompareEffect(() => {
-    if (
-      !!formikRef?.current &&
-      isMouseEventRecent(parcelLayerData?.e) &&
-      !!parcelLayerData?.data &&
-      !getIn(formikRef.current.values, `${nameSpace}.id`)
```

Comment:
allow duplicate requests so that data is refreshed and replaces the results of other unrelated searches.